src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -14,34 +15,42 @@ namespace Microsoft.DurableTask.Client.Grpc;
 /// <summary>
 /// Durable Task client implementation that uses gRPC to connect to a remote "sidecar" process.
 /// </summary>
-sealed class GrpcDurableTaskClient : DurableTaskClient
+public sealed class GrpcDurableTaskClient : DurableTaskClient
 {
     readonly ILogger logger;
     readonly TaskHubSidecarServiceClient sidecarClient;
-    readonly GrpcDurableTaskClientOptions grpcOptions;
+    readonly GrpcDurableTaskClientOptions options;
     readonly AsyncDisposable asyncDisposable;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="GrpcDurableTaskClient"/> class.
     /// </summary>
     /// <param name="name">The name of the client.</param>
-    /// <param name="options">The common client options.</param>
-    /// <param name="grpcOptions">The gRPC client options.</param>
+    /// <param name="options">The gRPC client options.</param>
     /// <param name="logger">The logger.</param>
+    [ActivatorUtilitiesConstructor]
     public GrpcDurableTaskClient(
-        string name,
-        DurableTaskClientOptions options,
-        IOptionsMonitor<GrpcDurableTaskClientOptions> grpcOptions,
```

Comment:
I guess it's nice that we have just one "options" here instead of two now. :)

Reply:
Yes, that was my primary goal here.